runtime/runtime/src/lib.rs

```diff
@@ -1267,7 +1275,12 @@ impl Runtime {
                 outgoing_receipts.push(receipt);
             }
 
+            // TODO(akashin): Why don't we have safe adds here? Do have some guarantee that there
```

Comment:
For reviewers: I'm double-checking that I'm not missing anything obvious here, as I think this code should use safe adds as well (and propagate any possible errors)

Reply:
Good find!
I suppose there is a limit for how many transactions are included, and there is a limit for how much gas each transaction can use. So I _think_ this is safe in practice right now.
But I would prefer we use checked add, or at least state very clearly in a comment why this cannot fail.

Reply:
Sounds good, sent https://github.com/near/nearcore/pull/8818 that changes this to `safe_add_gas`.